src/components/constants/Items.ts

```diff
@@ -408,6 +408,226 @@ export class Items {
         "Town Map": {
             "effect": "Shows your position in the Pokemon World",
             "category": "Key"
-        }
+        },
+        "HM01": {
+            "effect": "Cut",
```

Comment:
The effect is the move name, and in this case it cuts something. I can make a "move name": key if you want and have the effect be what using the item does aka (Teaches a Pokemon the *move name*).